docs/rfcs/rfc_0001_deductions_from_the_mission_statement.md

```diff
@@ -0,0 +1,111 @@
+
+
+## Key points from the mission statement
+
+> Imagine a unique social project that takes a web server and turns it into a multiplayer hub for 
+> a purposeful cause. The goal? To create a space where people come together, enjoy games or 
+> activities, and contribute to a shared fund. 
+> This fund isn’t just any fund – it’s a resource that automatically gets divided among the users of the server. 
+> So, the more you engage and participate, the more you receive from the fund. 
+> This innovative project not only brings people closer through fun and interaction but also 
+> enables them to collectively impact their community by distributing funds where they’re needed most.
+
+Technology wise, we extract the following points:
+- A web server.
+- Games and activities.
+- People can contribute to a shared fund.
+- The fund gets distributed among the users of the server.
+- A user's income from the fund is a function of their engagement.
+- Income is also distributed based on necessity.
+
+
+## A study of flows
+
+One key ambiguity in the mission statement is if value is expected to be internal or external to the server.
+We will study the internal case and study the external case separately
+
+### Internal flows of value
+
+A key item of the mission statement is the existence of a shared fund.
+This fund MAY receive value, and that value MUST flow to each user in the server eventually.
+
+Value flow from the shared fund to each user MUST be calculated according to a function.
+This function SHOULD take into account:
+- how much a given user has engaged with the server.
+- how much a given user needs value.
+- global engagement and global need of value.
+
+Users MUST have a private, in-server fund in which to store their value.
+
+### External flows of value
+
+If external value flows are desired, two more flows are possible:
+- Inward flows of value to the server  (for example, donations).
+- Outward flows of value from the server (for example, users withdrawing funds from their personal fund).
+
+#### Inward Flows of value
+
+The main inward flow of value is donations.
+
+#### Outward Flows of value
+
+Two main outward flows of value are considered:
+- Users withdrawing their funds.
+- The Shared Server Fund paying maintenance fees (such as electricity, hosting fees, etc.).
+
+This is easily achievable by integrating with existing cryptocurrencies.
+
+## Deep dive into fund sharing
+
+This section explores alternatives to fund distribution.
+Explored alternatives are:
+- Direct Split.
+- Fraction Sharing Split.
+- Fraction Sharing Split with Weighted Engagement.
+
+### Direct Split
+
+The simplest of splits. Split the Server Fund contents equally among the users.
+
+{equation}
+
+### Fraction Sharing Split
+
+At each time increment, only split a fraction of the Server Fund contents equally among the users.
+This allows for a more gradual transfer of funds among users.
+
+{equation}
+
+In time, this metric resembles a negative exponential.
+
```

Comment:
please do the math, i know you reeealy wanna use the term, i know the feeling <3
like no one stopping you, you can use it, just try^~^ and if issues lemme know and ill help

tbh i have no idea yet what you wanna use it for, but that shall not be stopping you^^